packages/plugins/test/mocha/common.ts

```diff
@@ -10,7 +10,6 @@ import {
   ConfigArtifacts,
   DeploymentInfo,
   ExecutionMode,
-  RawActionInput,
```

Comment:
The changes to this file stem from the PR's removal of the `RawActionInput` type, which were previously used to define create2 deployment actions for our TypeScript tests. We now use `AccountAccess` objects for this purpose